lib/daru/io/exporters/csv.rb

```diff
@@ -80,6 +80,7 @@ def write(path)
             contents.each { |content| csv << content }
             csv.close
           end
+          true
```

Comment:
Why is it necessary?

Reply:
I thought that this could act as a consistent means of checking whether the "writing into a file" actually took place. Else, the `File.write(...)` just returns the number of characters written - which may not mean much to the user/developer compared to a consistent (constant) `true`. 

It's not really necessary, but I thought this would be better. Let me know if this can be reverted.